ydb/public/sdk/cpp/client/ydb_topic/include/client.h

```diff
@@ -52,6 +56,9 @@ class TTopicClient {
     TAsyncStatus CommitOffset(const TString& path, ui64 partitionId, const TString& consumerName, ui64 offset,
         const TCommitOffsetSettings& settings = {});
 
+    TAsyncStatus RetryOperation(TOperationWithoutSessionFunc&& operation,
```

Comment:
client calls method which takes another client as argument

maybe make free function to handle retries of arbitrary operations?